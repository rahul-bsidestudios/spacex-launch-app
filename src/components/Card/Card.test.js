import React from "react";
import { shallow } from "enzyme";
import { Card } from "./index";

describe("Test cases for Card component", () => {
	const mockProps = {
		data: {
			flight_number: "1",
			mission_name: "Sat-1",
			launch_year: "2006",
			mission_id: [],
			launch_success: true,
			land_success: false,
			links: {
				mission_patch_small: "test.png"
			}
		}
	};
	const component = shallow(<Card { ...mockProps } />);
	
	test("Should match snapshot", () => {
		expect(component).toMatchSnapshot();
	});
});