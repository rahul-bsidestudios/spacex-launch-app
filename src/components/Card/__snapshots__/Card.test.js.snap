// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test cases for Card component Should match snapshot 1`] = `ShallowWrapper {}`;
